index.html

```diff
@@ -573,7 +573,7 @@ <h3 class="color-primary">Healme Harry</h3>
                 <span class="label label-primary pull-right">Caster</span>
                 <h3 class="color-warning">PhoenixChao</h3>
                 <p>PhoenixChao is a FFXIV/Variety Broadcaster. He’s an Astrologian main and a
-                midcore raider. He loves singing during his streams.</p>
+                midcore raider. He loves singing during his streams.</p> <br>
```

Comment:
Why do we need the line break here?

Reply:
Ooooo you can remove it. I was toying with what it would look like if the cards were the same height. Chao's is shorter because his sentence is shorter. I forgot to take it back out. I realized that once I scrolled to the bottom, all of the cards were different heights.